_posts/2023-11-09-in-progress-wildfly-feature-on-openshift.adoc

```diff
@@ -0,0 +1,276 @@
+---
+layout: post
+title: 'Trying Out An In-progress Feature for WildFly on OpenShift'
```

Comment:
I would  specify what the In-Progress feature is. For example:
`Trying out upcoming ability to add additional scope values when securing applications using OpenID Connect` or `Trying out upcoming feature of specifying additional scope values when securing applications using OpenID Connect`
WDYT?

Reply:
Right. I also think it would be good to explain what an in-progress feature is. 
Do you think it would be better to explain that in the synopsis and also on the first pagaraph? Just to keep the title more concise.

Reply:
@PrarthonaPaul I think you can name it "upcoming" instead of in-progress? That would not require an explanation, WDYT?